src/lib/xmlparse.rs

```diff
@@ -3565,9 +3615,8 @@ impl<'scf> XML_ParserStruct<'scf> {
             let mut next: *const c_char = buf.as_ptr();
             let mut tok = (*enc).xmlTok(XML_STATE::CONTENT, buf, &mut next);
             *eventEndPP = next;
-            let mut current_block_275: u64;
             match tok {
-                super::xmltok::XML_TOK::TRAILING_CR => {
```

Comment:
Future note: these are basically style changes, should probably have gone into a separate commit (maybe directly to `master`). You can leave them in this PR.